clickhouse-r2dbc/pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>clickhouse-java</artifactId>
+        <groupId>com.clickhouse</groupId>
+        <version>${revision}</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>clickhouse-r2dbc</artifactId>
+
+    <properties>
+        <revision>0.3.3-SNAPSHOT</revision>
+        <hikari-cp.version>4.0.3</hikari-cp.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <r2dbc-spi.version>1.0.0.RELEASE</r2dbc-spi.version>
+        <commons-lang3.version>3.12.0</commons-lang3.version>
+        <testcontainers.elasticsearch.version>1.16.3</testcontainers.elasticsearch.version>
+        <testcontainers.clickhouse.version>1.16.3</testcontainers.clickhouse.version>
+        <docker-java-api.version>3.2.13</docker-java-api.version>
+        <docker-java-transport-zerodep.version>3.2.13</docker-java-transport-zerodep.version>
+        <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
```

Comment:
This has been specified in parent pom.

Reply:
removed and only commons-lang3 property is moved to the parent pom.